### PR TITLE
Update What's New for October 14 - November 3

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Updates the What's New for new entries between October 14 - November 4.

## Affected DevDocs pages

-  https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+- description: Added information about two hot fixes available for Magento 2.3.6 and
+    Magento 2.4.1. See [Magento 2.3.x release information](https://devdocs.magento.com/guides/v2.3/release-notes/bk-release-notes.html)
+    and [Magento 2.4.x release information](https://devdocs.magento.com/guides/v2.4/release-notes/bk-release-notes.html).
+  versions: 2.3.4, 2.3.6
+  type: Major Update
+  date: October 29, 2020
+  link: https://github.com/magento/devdocs/pull/8150
+  contributor: jfrontain
+  profile: https://github.com/jfrontain
+- description: Updated Magento Cloud Docker instructions for adding custom PHP extensions.
```
Comment (shrielenee) [inline src/_data/whats-new.yml:18]: Can we just link to the updated doc in this first sentence?

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+  profile: https://github.com/Rykh
+- description: Specified that Magento supports the latest 1.x version of Composer
+    in the [system requirements](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html).
+  versions: 2.3.x, 2.4.x
+  type: Technical
+  date: October 28, 2020
+  link: https://github.com/magento/devdocs/pull/8142
+  contributor: jeff-matthews
+  profile: https://github.com/jeff-matthews
+- description: Updated the _Installation Guide_ to indicate that the `explicit_defaults_for_timestamp`
```
Comment (shrielenee) [inline src/_data/whats-new.yml:43]: I'm again not sure if this needs to be two sentences.

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+    setting for MySQL has been deprecated. See [Configuring the Magento database instance](https://devdocs.magento.com/guides/v2.4/install-gde/prereq/mysql.html#instgde-prereq-mysql-config).
+  versions: 2.x
+  type: Technical
+  date: October 28, 2020
+  link: https://github.com/magento/devdocs/pull/8137
+  contributor: BarnyShergold
+  profile: https://github.com/BarnyShergold
+- description: Added detailed information about the technical review guidelines, tests,
+    tools, and processes that Magento uses to verify and validate third-party extensions
+    submitted to the Magento Marketplace Developer Portal. See [Submit for Technical
```
Comment (shrielenee) [inline src/_data/whats-new.yml:53]: Again with multiple sentences? Do we need that?

File: src/_data/whats-new.yml [hunk 4]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+  contributor: dobooth
+  profile: https://github.com/dobooth
+- description: Updated the [Module Reference Guide](https://devdocs.magento.com/guides/v2.3/mrg/intro.html).
+  versions: 2.3.6
+  type: Major Update
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8057
+  contributor: dshevtsov
+  profile: https://github.com/dshevtsov
+- description: Added Magento Cloud Patches v1.0.8 release notes.
```
Comment (shrielenee) [inline src/_data/whats-new.yml:112]: Link here?

File: src/_data/whats-new.yml [hunk 5]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+    for store settings](https://devdocs.magento.com/cloud/live/sens-data-over.html#configuration-logic-flow)
+    topic in the _Cloud Guide_ that describes how Magento applies configuration settings
+    that have been defined in multiple contexts.
+  versions: 2.x
+  type: Major Update
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8031
+  contributor: BarnyShergold
+  profile: https://github.com/BarnyShergold
+- description: Published all documentation updates related to the Q4 2020 Commerce
```
Comment (shrielenee) [inline src/_data/whats-new.yml:129]: Link?

File: src/_data/whats-new.yml [hunk 6]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+    and Open Source releases.
+  versions: 2.3.6, 2.4.1
+  type: Major Update
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8051
+  contributor: jeff-matthews
+  profile: https://github.com/jeff-matthews
+- description: Updated reference documentation for Backward incompatible changes introduced
```
Comment (shrielenee) [inline src/_data/whats-new.yml:137]: Should "Backward incompatible" be capitalized here?

File: src/_data/whats-new.yml [hunk 7]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+    and [2.3.6](http://127.0.0.1:4000/guides/v2.3/release-notes/backward-incompatible-changes/reference.html#235---236)
+    releases.
+  versions: 2.3.6, 2.4.1
+  type: Major Update
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8054
+  contributor: devops-devdocs
+  profile: https://github.com/devops-devdocs
+- description: Updated `bin/magento` reference for [Commerce](https://devdocs.magento.com/guides/v2.4/reference/cli/magento-commerce.html)
+    and [OpenSource](https://devdocs.magento.com/guides/v2.4/reference/cli/magento.html).
```
Comment (shrielenee) [inline src/_data/whats-new.yml:148]: Is OpenSource supposed to be one word like this?

File: src/_data/whats-new.yml [hunk 8]
```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
...
+  versions: 2.4.1
+  type: Major Update
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8055
+  contributor: dshevtsov
+  profile: https://github.com/dshevtsov
+- description: Update list of required [PHP extensions](https://devdocs.magento.com/guides/v2.4/install-gde/prereq/php-settings.html),
```
Comment (shrielenee) [inline src/_data/whats-new.yml:155]: Needs to be in past tense, and I'm not sure the comma is needed.

File: src/_data/whats-new.yml [hunk 9]
```diff
@@ -162,6 +333,31 @@ entries:
   link: https://github.com/magento/devdocs/pull/7436
   contributor: keharper
   profile: https://github.com/keharper
+- description: Published release notes for Magento Cloud Patches version 1.0.7.
```
Comment (shrielenee) [inline src/_data/whats-new.yml:336]: link?

**Review by dobooth — DISMISSED** [review]

**Review by shrielenee — CHANGES_REQUESTED** [review]
I noticed some inconsistencies, which I've commented on. 

Another thing that gives me pause is the convention of not linking the updated topic in the first summary sentence. In these cases, it feels as if the first sentence summarizing the topic changes is sufficient to link from, instead of adding another "See XXX topic" sentence with the link to the revised page.

File: src/_data/whats-new.yml (hunk 5 above)
Comment (meker12) [reply src/_data/whats-new.yml:129]: There's 125 files here, so that's why there are no links.

**Review by shrielenee — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**